rdt/transformers/datetime.py

```diff
@@ -146,8 +143,14 @@ def _reverse_transform(self, data):
             pandas.Series
         """
         data = self._reverse_transform_helper(data)
+        datetime_data = pd.to_datetime(data)
+        if type(datetime_data) != pd.Series:
+            datetime_data = pd.Series(datetime_data)
```

Comment:
`to_datetime` will return a `Series` if `data` is a `Series`, otherwise it returns a `DatetimeIndex`. This becomes a problem on line 151 because `DatetimeIndex` can directly call `strftime` but `Series` can only do it through the `.dt` attribute. We also should make sure a series is always returned, so it's best just to convert anything that isn't a `Series` here.